### ISSUE TITLE
Watched Lecture Videos from Google Data Analytics course (06/21)

### ISSUE THREAD

**Issue by anitabe404** [studydash/cards#129]
## Motivation/Problem Statement 💭❓:
Keep calm, code on.

---

## Today's Goal/Solution 🥅:
Data Analytics

---

## Result 🍓🍌🍉:
Watched videos and took some quizzes in the Prepare Data for Exploration course. The videos gave a high-level overview of databases and metadata. 

---

## Observations & Next Steps 🔭👣:
I'm wading into new territory, so it'll be interesting to see how the Google course ties all of these concepts together during the labs and other coursework. So far I like Dr. Severance's explanation of databases better (from the Using Databases with Python course). However, that might just be the nerd in me. The Google course provided interesting insights even though it was less technical.

---

## References 🔗:
Link any useful resources you may have found throughout the day.

---

## Extra/Fun (Optional) 🎈🎉🥳:
I went grocery shopping today. I hadn't gotten away from the house in a while. I'm beat today though! I had to keep my learning session short and I'm going to be early (for me).

**Comment by r002** [comment]
This is awesome!  Is this the [Google Data Analytics certificate](https://grow.google/dataanalytics) that you're currently pursuing?  Data science and analytics is one of my hobbies too!  In fact, I've been recently kicking around a data analytics dashboard idea for tracking gun violence in the USA.  While I totally understand why some people in America want to own guns (it's pretty rural in many places of the country; if a burglary were to happen, the police would never arrive on time!), I am also **_extremely_** concerned with the horrific number of gun deaths we [suffer](https://www.vox.com/2015/10/1/18000444/gun-crime-us-foreign-countries) in the US.  😦😧☹

My idea's still inchoate but the first step I'm going to do is build a web scraper (that uses [go-colly](https://github.com/gocolly/colly)) that scrapes the [gunviolencearchive.org page](https://www.gunviolencearchive.org/last-72-hours) for data.  I think building the dashboard will be a fun nights-and-weekend project and also help me better learn Go!  (And if everything goes right, may even be able to raise some awareness in this area for general folks too!)

---

> _From [Vox](https://www.vox.com/policy-and-politics/2015/12/4/9850572/gun-control-us-japan-switzerland-uk-canada):_

![image](https://user-images.githubusercontent.com/45280066/122968647-c2e0e200-d359-11eb-9bfa-c6cdc48f7715.png)

**Comment by anitabe404** [comment]
@r002 Yes, that's the certificate that I'm working towards. I actually started it before I started doing the Python for Everybody specialization. I ended up progressing through the Python course faster because I found it more engaging. However, I committed to completing both certs, so I still work on the Google one. It's getting more interesting.

Yes, this is an interesting thing to analyze. I too worry about our incidence of gun deaths and gun violence in general. In addition to being a complex thing to analyze in general, the data collection aspect of it makes it even more challenging. A lot of the data is in disparate systems that aren't linked together. Additionally, there can be inaccuracies in the data that's not easy to detect if you're not able to discuss the context with the entity that collected it. Another thing is looking at who collected the data and why. Are they actually following the data or are they letting the data follow their beliefs? There's so much to it.

**Comment by r002** [comment]
Totally agree with all of your points!  I think I've mentioned this before but last year I detoured into several months of exploring data analytics.  It was fun and I learned many different tools and methodologies (SimPy, statistics, distributions, Excel can do Monte Carlo simulations!, etc); but the one takeaway I really learned from that experience was --for me at least-- to pursue a career in data science, I'd really need to be onboard with the **_mission_** of the organization.  Like you'd mentioned, there's so much that goes into the actual collection/wrangling of the data itself.  And from the little I saw, I'm pretty convinced that you can spin the data any way you please to tell **_any_** kind of story based on one's agenda.

Hilariously, I'd entered the field thinking, "let's follow the data wherever it may lead!" but I now believe that's an enormously naïve sentiment.  So my general attitude towards data science now is to employ it as a very specific tool of hyper-narrow inquiry.  For example, I've been curious about gun violence in America.  But more specifically:  I'm curious about what places in America does gun violence happen most?  What neighborhoods in what towns/cities?  And specifically, what the socioeconomic contexts are in those environments.  How many incidents are mental-health related or crime-related?  It's all very interesting-- can't wait to launch into the project!

**Comment by anitabe404** [comment]
> And from the little I saw, I'm pretty convinced that you can spin the data any way you please to tell **_any_** kind of story based on one's agenda.

Yes. And I think it's even trickier today because it's so easy to create and disseminate biased or even false conclusions based on data. And because it's easy to get lost in the fancy charts, it's not always apparent. As I've been looking through Kaggle, which has open datasets, my first question is always, "where did this data come from," and so many times it has been data from some other source and then I have to go there and ask the same question.

It's a bit disconcerting especially since we live in a world with increasing ever-increasing amounts of data. So much so that we're turning to machine learning and quantum computing to process it. Where does that leave the average person in being able to evaluate data? When you're going up against AI hosted on quantum computing machines. 😂

It sounds like an interesting question. If you actually delve into and need a partner or someone to red team with, let me know.

**Comment by r002** [comment]
@anitabe404 Cool!  Thank you for offering to help with the data project! 🙏 Since it looks like you're increasingly exploring and learning about Python/data science/analytics, this could be a chance to build something neat!  I'll definitely keep you posted as I figure out what it is I actually want to investigate.  Haha, nowadays, my big challenge is definitely staying focused and not chasing too many shiny objects-- I'm just curious about so many things; there never seems to be enough time! 😅
